docs/scos/dev/feature-integration-guides/202204.0/product-bundles-cart-feature-integration.md

```diff
@@ -63,3 +63,135 @@ Add several regular products and product bundles to cart.
+use Spryker\Zed\ProductBundle\Communication\Plugin\Cart\UnfoldBundlesToUnitedItemsItemExpanderPlugin;
+
+class CartDependencyProvider extends SprykerCartDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Spryker\Zed\CartExtension\Dependency\Plugin\ItemExpanderPluginInterface>
+     */
+    protected function getExpanderPlugins(Container $container): array
+    {
+        return [
+            // new ExpandBundleItemsPlugin(),
+            new UnfoldBundlesToUnitedItemsItemExpanderPlugin(),
+        ];
+    }
+    
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Spryker\Zed\CartExtension\Dependency\Plugin\CartOperationPostSavePluginInterface>
+     */
+    protected function getPostSavePlugins(Container $container): array
+    {
+        return [
+            // new CartPostSaveUpdateBundlesPlugin(),
+            new RefreshBundlesWithUnitedItemsCartOperationPostSavePlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Client/Cart/CartDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Client\Cart;
+
+use Spryker\Client\Cart\CartDependencyProvider as SprykerCartDependencyProvider;
+use Spryker\Client\ProductBundle\Plugin\Cart\ReplaceBundlesWithUnitedItemsCartChangeRequestExpanderPlugin;
+
+class CartDependencyProvider extends SprykerCartDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Client\CartExtension\Dependency\Plugin\CartChangeRequestExpanderPluginInterface>
+     */
+    protected function getRemoveItemsRequestExpanderPlugins(): array
+    {
+        return [
+            // new RemoveBundleChangeRequestExpanderPlugin(),
+            new ReplaceBundlesWithUnitedItemsCartChangeRequestExpanderPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/PersistentCart/PersistentCartDependencyProvider.php***
+
+```php
+<?php
+
+namespace Pyz\Zed\PersistentCart;
+
+use Spryker\Zed\PersistentCart\PersistentCartDependencyProvider as SprykerPersistentCartDependencyProvider;
+use Spryker\Zed\ProductBundle\Communication\Plugin\PersistentCart\ReplaceBundlesWithUnitedItemsCartChangeRequestExpandPlugin;
+
+class PersistentCartDependencyProvider extends SprykerPersistentCartDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\PersistentCartExtension\Dependency\Plugin\CartChangeRequestExpandPluginInterface>
+     */
+    protected function getRemoveItemsRequestExpanderPlugins(): array
+    {
+        return [
+            // new RemoveBundleChangeRequestExpanderPlugin(),
+            new ReplaceBundlesWithUnitedItemsCartChangeRequestExpandPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Add a product bundle to cart, try to increase its quantity to a large amount, i.e. 1000 items. Then try to decrease the 
+quantity. Make sure that increase and decrease operations are performed without a significant delay and do not fail 
```

Comment:
Why are the text in multiple lines between 179 and 181?

Reply:
Fixed. I thought that for markdown it is not important where to put line endings since in the reading mode they don't count anyway, but now @VadymSachenko suggested me to add a carriage return after a point at the end of a sentence.